locale/cy_GB.UTF-8/LC_MESSAGES/TheyWorkForYou.po

```diff
@@ -525,7 +525,17 @@ msgstr "Cwestiynau Maer Llundain"
 msgid "Senedd debates"
 msgstr "Dadleuon y Senedd"
 
-#: scripts/alertmailer.php:278
```

Comment:
This seems to have been inserted in an odd place, we've lost this comment line for the "as a teller"

Reply:
So at this point  the comment lines that are generated are a big diff from what's committed. I think I should just do another commit catching it up, having separated out the main change this time.

Reply:
Added that

Reply:
The source .po file should also be committed then?
Thanks, yes, you shouldn't be manually editing .po files :)